pages/Messages/Signatures.md

```diff
@@ -51,28 +53,28 @@ For example, given the following DSNP broadcast message:
 This would be the expected serialization:
 
 ```
-contentHash0x67890fromAddress0x12345urihttps://www.projectliberty.io/
+\x19Ethereum Signed Message:\n\x69contentHash0x67890fromAddress0x12345urihttps://www.projectliberty.io/
 ```
 
 This would be the expected hash:
 
 ```
-0x45c42591e0154088325055f26664002bf05f211db9e5d7c7bfc588dc309698e0
+0x4c233fc1d71f92e4b8b7408291d45b5aa2ec3689d14091abc9207c4eca3886ad
```

Comment:
I get `0x6ad0d59e6d5f7aaee5998e5d584d8c55a3c01f09bf0f5b4c49b2399eca22a82a` when running this through manually and through https://web3js.readthedocs.io/en/v1.3.4/web3-eth-accounts.html?highlight=recover#hashmessage which should result in the same hash.

I think it might be that the length is supposed to be in decimal not hex.

Reply:
Fixed